index.html

```diff
@@ -26,16 +26,131 @@ <h1 class="primaryHeading">
         <p class="mainContent">
             I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
         </p>
-
         <a href="#" class="connect">Let's Connect</a>
         <ul class="icons">
-           <li><a href="#" class="iconList"><i class="fa-brands fa-twitter"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-linkedin-in"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-github"></i></a></li>
-           <li><a href="#" class="iconList"><i class="fa-brands fa-instagram"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="#" class="iconList"><i class="fa-brands fa-instagram"></i></a></li>
         </ul>
 
     </main>
-    
+
+    <section class="workSection">  
+        <div class="card">
+            <div class="image2 img"></div>
+            <h1 class="tonic">Tonic</h1>
+
+            <div class="divList">
+                <p class="canopy">canopy</p>
+                <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+                <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+            </div>
+
+            <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="language">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javacript</li>
+            </ul>
+            <a href="#" class="seeProject">See Project</a>
+        </div>
```

Comment:
- [ ] It would be beautiful to add background images to the cards. I am sure you will also like to have it displayed beautifully just as in the Figma design.
### Figma design
![image](https://user-images.githubusercontent.com/86200356/198748910-e6bc2ddf-bd4f-4a5b-a67e-6c54971ed405.png)
### Your design
![image](https://user-images.githubusercontent.com/86200356/198748942-d113ab54-a810-4878-9568-5830f4e91e4f.png)